.gitignore

```diff
@@ -11,6 +11,7 @@ lib/*
 ./config.json
 src/test/data/sandbox/
 preferences.json
+config.json
```

Comment:
Should not gitignore config.json since the configuration file for reposense is named as that as well

Reply:
but then i shouldn't commit the config.json file right